src/vm/eventtracepriv.h

```diff
@@ -291,6 +291,10 @@ class BulkTypeEventLogger
     // List of types we've batched.
     BulkTypeValue m_rgBulkTypeValues[kMaxCountTypeValues];
 
+#ifdef FEATURE_PAL    
+    BYTE m_BulkTypeEventBuffer[65536];
```

Comment:
what's preventing us from exceeding the 64k buffer? since I don't see any checks in its usage in FireBulkTypeEvent.